GSA/digitalgov.gov
# Fix image processing

**Summary**



This PR implements the following **changes:**

* [ ] remove the `set-env` task from gulpfile.js (was not working) and references to it in package.json
* [ ] sets the node_env to production when running on federalist
* [ ] add a check for the `development` environment before loading image tasks
* [ ] adds a watch task to monitor the `content/images/_inbox` folder for changes; when a change is made (image added) it will kick of the `img` task to process and upload the image
* [ ] add `dist` folder to the .gitignore file so we're not asked to commit any changes that get compiled there
* [ ] update uswds.js and the browsers list that was breaking due to recent changes on [uswds-gulp](https://github.com/uswds/uswds-gulp/commit/9b33185ac85f0a6776433aecce6d10f92fd71498)

Documentation has also been updated: 
https://github.com/GSA/digitalgov.gov/wiki/Adding-Images

Testing
From your project repository, bring in the changes to test:
first be sure you have the lastest production build
* git pull origin master
* git fetch origin  

then checkout this branch  
* git checkout -b sc_fix-gulp-img origin/sc_fix-gulp-img
and merge it with master so you're testing the most up-to-date version
* git merge master

and spin the platform up locally and test with images
* run `npm start`
* add a test image to the `content/images/_inbox` folder
* verify that the img script ran and the appropriate files were created
